typescript/phoenix-sdk/src/market.ts

```diff
@@ -94,53 +91,51 @@ export class Market {
    * @param marketAddress The `PublicKey` of the market account
    */
   static async load({
-    connection,
     address,
+    buffer,
+    tokenList,
   }: {
-    connection: Connection;
     address: PublicKey;
+    buffer: Buffer;
+    tokenList: TokenConfig[];
```

Comment:
does the market config not tell us which tokens belong to the market? we should be able to pass in a base token config and a quote token config directly rather than finding them in the token list

Reply:
(upstream of here)